examples/timezone/README.md

```diff
@@ -0,0 +1,7 @@
+# Timezone
+
+This example contains a complete `cookbooks/` directory that you can use on the
+stable-v5 stack to change your server's timezone.
+
+See [cookbooks/custom-timezone](cookbooks/custom-timezone/README.md) for complete
```

Comment:
This is relative path from this README which is already under examples/timezone. See https://raw.githubusercontent.com/engineyard/ey-cookbooks-stable-v5/next-release/examples/sidekiq/README.md. Verified that the relative path works.